### PR TITLE
update ziti-sdk-c

### PR DESCRIPTION
**Pull request by ekoby**

### CODE REVIEW

**Comment by dovholuknf** [comment]
pls don't merge this till after the new year to be safe

**Review by smilindave26 — APPROVED** [review]

File: lib/ziti_tunnel_cbs.c
Comment (scareything) [inline lib/ziti_tunnel_cbs.c:460]: Can the data_cb be called after ziti_close is called?

If so we would need to move the free'ing / closing of any io context members to the close_cb.

**Review by scareything — APPROVED** [review]

File: lib/ziti_tunnel_cbs.c
Comment (ekoby) [reply lib/ziti_tunnel_cbs.c:460]: we probably should move some of the cleanup into close_cb, however I didn't want to do it without evaluating the rest of the client lifecycle

**Pull request merged**
